cohort-evaluator-spark/pom.xml

```diff
@@ -182,6 +182,12 @@
 		<dependency>
 			<groupId>org.apache.hadoop</groupId>
 			<artifactId>hadoop-common</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>io.netty</groupId>
+					<artifactId>netty</artifactId>
+				</exclusion>
+			</exclusions>
```

Comment:
Did you consider putting this exclusion in the parent POM? If we don't want netty with Hadoop, we probably don't want it everywhere we happen to use the hadoop dependency.

Reply:
Yes, meant for it to go in the parent. Good catch.